client/app/validators/required-if-selected.js

```diff
@@ -0,0 +1,30 @@
+import {
+  validatePresence,
+} from 'ember-changeset-validations/validators';
+
+export default function validatePresenceIf(options) {
```

Comment:
@allthesignals can you explain that this function is doing?

Reply:
Sorry, yes — it conditionally validates the presence of a field. For example, in the form, if I've selected "yes" to one question, and it asks a follow-up question, this validation will make that follow-up question _required_ but only if "yes" is selected.

Reply:
Gotchya, thank you -- that's helpful and makes sense